docs/source/common-configuration-patterns.md

```diff
@@ -9,8 +9,6 @@ Invoking `eogrow-template "eogrow.pipelines.zipmap.ZipMapPipeline" "zipmap.json"
 {
   "pipeline": "eogrow.pipelines.zipmap.ZipMapPipeline",
   "pipeline_name": "<< Optional[str] >>",
-  "workers": "<< 1 : int >>",
```

Comment:
This would now be controlled via the ray remote kwargs, right?

Reply:
it might be cumbersome in the future to provide cpu workers via ray resources. we might need to think about some more convenient ways to set number of workers or something.

Reply:
In the past we had the option to set the amount of CPU workers via the `workers` parameter. This parameter is not deprecated, but controlling CPU workers can still be done via the `ray_remote_kwargs`. I just wanted to point out that it might be a bit unintuitive to provide CPU workers that way in case of unskilled users. What do you think?

Reply:
i don't think performance tweaking is meant for unskilled users. They can also set up the local ray cluster with restricted CPUs.